### ISSUE TITLE
Light theme?

### ISSUE THREAD

**Issue by hmaarrfk** [ColinDuquesnoy/QDarkStyleSheet#240]
I know the whole point of this is to have a good looking dark theme, but many of the style changes you've done look quite good (increasing the size of certain elements and whatnot), so I was wondering if there was any interest in adding a "light" theme that would match the dark theme you've created?

Is there one already available? Maybe I just can't find it.

Thanks for the consideration!

**Comment by ccordoba12** [comment]
We're thinking to create a light theme based on this one for Spyder 5. I hope we have enough time for that.

**Comment by dpizetta** [comment]
Just to document, Spyder team is working on this: https://github.com/ColinDuquesnoy/QDarkStyleSheet/issues/252

**Comment by dpizetta** [comment]
Implemented in #268

**Comment by mgrojo** [comment]
I'm not sure if I should open a new issue or not, but I'll do it if I'm said to.

I've tried to add the new light style to [DB Browser for SQLite](https://github.com/sqlitebrowser/sqlitebrowser). I want to provide three options: follow the desktop style, dark style and light style. I cannot add these two options with qdarkstyle at the same time. My impression is that the Qt or cmake generation for the resources are overlapped due to usage of the same file name for both savours.  This is what I've changed in [CMakeLists.tst]( https://github.com/sqlitebrowser/sqlitebrowser/blob/c5199f559d2e734075af85faed2a00ed334cdcd0/CMakeLists.txt#L)277 to include both styles:

```diff
diff --git a/CMakeLists.txt b/CMakeLists.txt
index 91573df6..22b1dbca 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -274,7 +274,8 @@ set(SQLB_RESOURCES
        src/translations/flags/flags.qrc
        src/translations/translations.qrc
        src/certs/CaCerts.qrc
-       src/qdarkstyle/style.qrc
+       src/qdarkstyle/light/style.qrc
+       src/qdarkstyle/dark/style.qrc
 )
 
 set(SQLB_MISC
```

Only the last listed qrc can be used, probably because this file is generated: `qrc_style.cpp`

Which is the recommended way to include both? The instructions in the README have not been updated, by the way.
